app/Journalbeat/doc/README.head.md

```diff
@@ -0,0 +1,8 @@
+Monitoring of [Journalbeat](https://github.com/mheese/journalbeat/).
+
+## Usage
```

Comment:
Consider moving the usage section to the docs template.

Reply:
Removed and updated with the help of the new default usage generation (#74).